CHANGELOG.md

```diff
@@ -1,6 +1,150 @@
 Changelog
 =========
 
+0.8.0
+-----
+
+**Date:** 2024-12-XX
+
+Key highlights for this release are a new atomic model FLYonPIC 2.0, the shadowgraphy plugin with advanced diagnostics for laser-plasma interactions,
+enhanced OpenPMD functionality, and support for the RISC-V ecosystem.
+As usual, this release features fixes of PIConGPU and extensions and clarifications of the documentation.
+This is the last release with C++17 support before we switch to C++20.
+
+**Bug Fixes:**
+- PIC:
+    - fix C++20 compile #5015
+    - fix Gaussian and Dispersive pulse #4889
+- PMacc:
+  - Fix null parameter in MPI_Init_thread #5230
+    - fix C++20 compiler #5015
+    - fix `atomicAllInc()` #4825
+    - avoid using uninitialized `pmacc::memory::Array` #4789
+- other:
+    - fix wrong conversion factor #5017
+    - correct spack options for boost #4780
+    - fix potential deadlock #4755
+- plugins:
+    - remove IDProvider from normal OpenPMD output #5193
+    - set main rank correctly in the binning plugin #4997
+    - fix incorrect data type #4992
+- tools:
+    - fix slow tbg substitution #4905
+    - fix growth rate and charge conservation test tools #4834
+ 
+**Documentation:**
+- PIC:
+    - replace cupla fully by alpaka #4838
+- PMacc:
+    - replace cupla fully by alpaka #4838
+- other:
+    - topic remove cuda11.3 and g++9/CUDA support #5010
+    - allow multiple gaussian density profiles with differing density #4981
+    - correct spack options for boost #4780
+    - RISC-V support #4712
+- plugins:
+    - add shadowgraphy plugin #4868
+    - remove deactivation of ISAAC with HIP #4821
+    - change class and function names in radPlugin to camelCase #4734
+    - add calorimeter python module #4697
+- tools:
+    - pre-commit #4792
+    - remove hemera k80 and k20 partition #4779
+    - add calorimeter python module #4697
+
+**Features:**
+- PIC:
+    - BSI minimal implementation #5205
```

Comment:
```suggestion
```
this is a FLYoPIC specific feature and should not mentioned separately

Reply:
added to the flyonpic point